package.json

```diff
@@ -121,5 +121,6 @@
     "extends": [
       "@commitlint/config-conventional"
     ]
-  }
+  },
+  "version": "0.0.0"
```

Comment:
Does new lerna add this version property in main package? Is it needed?

Reply:
lerna seemed to add it when i smoke tested using `yarn run version`

Reply:
Strange, it didn't do that before. but that shouldn't break the release anyway so it's fine